### PR TITLE
add blog post about Okta Plugin

### PR DESCRIPTION
**Pull request by Issif**
Signed-off-by: Issif <issif+github@gadz.org>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area blog

> /area documentation

> /area videos

**What this PR does / why we need it**:

Blog Post about Okta Plugin for Falco.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

WIP until https://github.com/falcosecurity/plugins/pull/83 and https://github.com/falcosecurity/falco/pull/1955 are merged

### CODE REVIEW

**Review by mstemm — APPROVED** [review]

File: content/en/blog/falco-okta-plugin.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+Falco has always excelled as a threat detection engine observing activity on Linux servers and Kubernetes clusters. Since Falco 0.31 it is also possible to **collect events from sources other than Kernel Syscalls and Kubernetes Audit logs**. The number of sources can be extended to cover any stream of events. For more information read our blogs posts about it:
+
+* [Falco Plugins Early Access](https://falco.org/blog/falco-plugins-early-access/)
+* [Falco 0.31.0 a.k.a. "the Gyrfalcon"](https://falco.org/blog/falco-0-31-0/)
+* [Announcing Plugins and Cloud Security with Falco](https://falco.org/blog/falco-announcing-plugins/)
+* [Extend Falco inputs by creating a Plugin: the basics](https://falco.org/blog/extend-falco-inputs-with-a-plugin-the-basics)
+
+In this blog post, we'll introduce a new plugin created by the Falco Team to collect [Okta Log Events](https://developer.okta.com/docs/reference/api/system-log/) and be able to trigger alerts whenever suspicious events are detected.
```
Comment (jasondellaluce) [inline content/en/blog/falco-okta-plugin.md:19]:
```suggestion
In this blog post, we'll introduce a new plugin created by the Falco Authors to collect [Okta Log Events](https://developer.okta.com/docs/reference/api/system-log/) and be able to trigger alerts whenever suspicious events are detected.
```

File: content/en/blog/falco-okta-plugin.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+| `okta.target.user.alternateid`  | string | Target User Alternate ID        |
+| `okta.target.user.name`         | string | Target User Name                |
+| `okta.target.group.id`          | string | Target Group ID                 |
+| `okta.target.group.alternateid` | string | Target Group Alternate ID       |
+| `okta.target.group.name`        | string | Target Group Name               |
+
+Okta Security Team also proposes a list of event types of interest:
+
+
+* User Events
```
Comment (jasondellaluce) [inline content/en/blog/falco-okta-plugin.md:116]:
```suggestion
#### User Events
```

File: content/en/blog/falco-okta-plugin.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+|eventType eq "user.session.end"|User logging out|
+|eventType eq “policy.evaluate_sign_on”|Sign in policy evaluation|
+|eventType eq “user.account.lock”|Okta user locked out|
+|eventType sw "user.authentication.auth"|All types of Auth events, covering MFA, AD, Radius, etc|
+|eventType eq "user.account.update_password"|User changing password|
+|eventType eq "user.authentication.sso"|User accesing app via single sign on|
+|eventType eq "user.authentication.auth_via_mfa"|MFA challenge|
+|eventType eq "user.mfa.factor.update"|User changing MFA factors|
+
+* Okta Events
```
Comment (jasondellaluce) [inline content/en/blog/falco-okta-plugin.md:130]:
```suggestion
#### Okta Events
```

**Review by jasondellaluce — COMMENTED** [review]
Left you few suggestion, not super strong about them. I see this is WIP until we have https://github.com/falcosecurity/falco/pull/1955 merged, so I'm gonna wait for it before approving.

**Comment by Issif** [comment]
@jasondellaluce as I'm linking the PR, I think we can release this blog post anyway, I don't consider it as a WIP anymore because the plugin itself has been released

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**